parsing_scripts/load_initial_depends.py

```diff
@@ -0,0 +1,69 @@
+import os, json
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "MacPorts.settings")
+
+import django
+
+django.setup()
+
+from ports.models import Dependency, Port
+
+
+def load_depends(list_of_dependencies, type_of_dependency, port):
+    dependency = Dependency()
+    dependencies = []
+    dependency.type = type_of_dependency
+    dependency.port_name = port
+    for depends in list_of_dependencies:
+        dependencies.append(Port.objects.get(name__exact=depends.rsplit(':', 1)[-1]))
+    dependency.save()
+    dependency.dependencies.add(*dependencies)
+
+
+with open('portindex.json', "r") as file:
+    ports = json.load(file)
+
+i = 0
+fixtures = []
+for port in ports:
+    iport = Port.objects.get(name__iexact=port['name'])
+    # Update Dependencies for the port
+    try:
+        load_depends(port['depends_lib'], "lib", iport)
+    except KeyError:
+        pass
```

Comment:
You should check whether the key exists instead of catching exceptions here. Or, even better, since you have the function already, why not pass the full port to the function, and then the function checks whether that key exists.

You could easily call the function simply as
```python
load_depends(port, 'lib', iport)
```
instead of all the four lines combined.

Further, you could actually squeeze the full code into two lines, with something like
```python
for dependency_type in ['lib', 'extract', 'run', ...]:
    load_depends(port, dependency_type, iport)
```